### ISSUE TITLE
Error decoding markdown files

### ISSUE THREAD

**Issue by vidartf** [minrk/pytest-check-links#2]
On Windows 10, conda install:
```
C:\dev\jupyterlab [master ≡]> py.test --check-links -k .md .
============================= test session starts =============================
platform win32 -- Python 3.5.2, pytest-3.0.6, py-1.4.32, pluggy-0.4.0
plugins: tornado-0.4.5, timeout-1.2.0, cov-2.4.0, check-links-0.0.2, nbval-0.4.1
collected 120 items / 3 errors

=================================== ERRORS ====================================
______________________ ERROR collecting design/about.md _______________________
C:\Miniconda3\lib\site-packages\pytest_check_links\plugin.py:61: in collect
    html = self._html_from_markdown()
C:\Miniconda3\lib\site-packages\pytest_check_links\plugin.py:33: in _html_from_markdown
    markdown = f.read()
C:\Miniconda3\lib\encodings\cp1252.py:23: in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
E   UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 2538: character maps to <undefined>
```

**Comment by vidartf** [comment]
Fixed in #1 .

**Comment by minrk** [comment]
Released as 0.1.0
